tests/alembic/revisions/test_08894082221a_replace_collection_name_with_observer_.py

```
@@ -0,0 +1,361 @@
+
+
+@register_before_upgrade("08894082221a")
+def prepare_observer_observed(verifier: MigrationVerifier) -> None:
+    schema = verifier.schema
+    conn = verifier.conn
+
+    # Create workspace
+    conn.execute(
+        text(
+            f'INSERT INTO "{schema}"."workspaces" ("id", "name") '
+            + "VALUES (:workspace_id, :workspace_name)"
```

Ruff (0.14.1)

Explicitly concatenated string should be implicitly concatenated

Remove redundant '+' operator to implicitly concatenate

(ISC003)